packages/components/psammead-bulletin/package.json

```
@@ -23,11 +23,12 @@
     "@bbc/psammead-styles": "^4.3.0",
     "@bbc/psammead-assets": "^2.13.0",
     "@bbc/psammead-visually-hidden-text": "^1.2.3",
-    "@bbc/psammead-story-promo": "^5.1.0"
+    "@bbc/psammead-story-promo": "^5.1.0",
+    "@bbc/psammead-live-label": "^0.1.0-alpha.2"
```

Should this use the new version?
```suggestion
    "@bbc/psammead-live-label": "^1.0.0"
```

---

Forgot to bump it there as well, thanks!